src/saf/collect/file.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+"""
+A file collector plugin.
+"""
+from __future__ import annotations
+
+import asyncio
+import logging
+import os
+import typing
+from typing import AsyncIterator
+from typing import Type
+
+if typing.TYPE_CHECKING:
+    import pathlib
```

Comment:
```suggestion
import pathlib
import asyncio
import logging
import os
import typing
from typing import AsyncIterator
from typing import Type
```
This will fix the docs build

Reply:
https://github.com/saltstack/salt-analytics-framework/blob/main/pyproject.toml#L105